src/clj/game/core/rules.clj

```diff
@@ -311,6 +311,7 @@
 (defn- resolve-trash
   [state side eid {:keys [zone type] :as card}
    {:keys [unpreventable cause keep-server-alive suppress-event] :as args} & targets]
+  (when (:installed card) (swap! state update-in [side :register :trashed-installed] (fnil inc 0)))
```

Comment:
Note that Corp cards never have `(:installed true)`. So if you want this to work equally well for both sides, it should be `(installed? card)` and you'll have to declare `installed?` up at the top of this file.